resources/views/user/show.blade.php

```diff
@@ -0,0 +1,76 @@
+@extends('layouts.dashboard')
+
+@section('title', 'Show User')
+
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12 col-sm-12 col-md-10 col-lg-10 col-xs-offset-0 col-sm-offset-0 col-md-offset-1 col-lg-offset-1">
+            <div class="panel panel-primary">
+                <div class="panel-heading">
+                    <h3 class="panel-title">User information</h3>
+                </div>
+                <div class="panel-body">
+                    <div class="row">
+                        <div class="col-md-3 col-lg-3 hidden-xs hidden-sm">
+                            <img class="img-circle"
+                                 src="https://lh5.googleusercontent.com/-b0-k99FZlyE/AAAAAAAAAAI/AAAAAAAAAAA/eu7opA4byxI/photo.jpg?sz=100"
+                                 alt="User Pic">
+                        </div>
+                        <div class="col-xs-2 col-sm-2 hidden-md hidden-lg">
+                            <img class="img-circle"
+                                 src="https://lh5.googleusercontent.com/-b0-k99FZlyE/AAAAAAAAAAI/AAAAAAAAAAA/eu7opA4byxI/photo.jpg?sz=50"
+                                 alt="User Pic">
+                        </div>
+                        <div class="col-md-9 col-lg-9 hidden-xs hidden-sm">
+                            <strong>{{ $user->name }}</strong><br>
+                            <table class="table table-user-information">
+                                <tbody>
+                                <tr>
+                                    <td>User level:</td>
+                                    <td>{{ $user->role }}</td>
```

Comment:
Can you replace the role id with the actual name or display them in combination like on the manage users page like Admin (3).

Reply:
fixed in commit 28bda35
note the new user model method